assets/stylesheets/sass/main/main.scss

```diff
@@ -259,17 +300,30 @@ figure {
     font-size: 1.3em;
   }
 
+  div {
+    height: 30px;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets